src/lib/route-broadcaster.js

```diff
@@ -0,0 +1,126 @@
+'use strict'
+
+const co = require('co')
+const defer = require('co-defer')
+const _ = require('lodash')
+const request = require('co-request')
+const BROADCAST_INTERVAL = 30 * 1000 // milliseconds
+
+class RouteBroadcaster {
+  /**
+   * @param {RoutingTables} routingTables
+   * @param {Backend} backend
+   * @param {Object} config
+   * @param {Object} config.ledgerCredentials
+   * @param {Object} config.tradingPairs
+   * @param {Number} config.minMessageWindow
+   */
+  constructor (routingTables, backend, config) {
+    this.baseURI = routingTables.baseURI
+    this.routingTables = routingTables
+    this.backend = backend
+    this.ledgerCredentials = config.ledgerCredentials
+    this.tradingPairs = config.tradingPairs
+    this.minMessageWindow = config.minMessageWindow
+    this.adjacentConnectors = {}
+    this.adjacentLedgers = {}
+    for (const pair of config.tradingPairs) {
+      const destinationLedger = pair[1].split('@')[1]
+      this.adjacentLedgers[destinationLedger] = true
+    }
+  }
+
+  * start () {
+    yield this.crawlLedgers()
+    yield this.reloadLocalRoutes()
+    yield this.broadcast()
+    setInterval(() => this.routingTables.removeExpiredRoutes(), 1000)
```

Comment:
Seems like this interval should be configurable and/or have a default value defined at the top of the file